models/book.py

```diff
@@ -1,17 +1,23 @@
-from sqlalchemy import Column, Enum, ForeignKey, Integer, String
-from sqlalchemy.orm import backref, relationship
-
-import utils.db
 from models.book_genre import BookGenre
+from resources import db
 
 
-class Book(utils.db.Base):
+class Book(db.Model):
     __tablename__ = "book"
-    id = Column(Integer, ForeignKey("library_item.id"), primary_key=True)
-    library_item = relationship("LibraryItem", backref=backref("book", uselist=False))
-    title = Column(String(50))
-    author = Column(String(60))
-    genre = Column(Enum(BookGenre))
+    id = db.Column(db.Integer, db.ForeignKey("library_item.id"), primary_key=True)
+    library_item = db.relationship(
+        "LibraryItem", backref=db.backref("book", uselist=False, cascade="all,delete")
+    )
+    title = db.Column(db.String(50), unique=True)
+    author = db.Column(db.String(60))
+    genre = db.Column(db.Enum(BookGenre))
 
     def __repr__(self):
         return str(self.__dict__)
+
+    @staticmethod
+    def from_dict(body: dict):
```

Comment:
Better to use .get("key", None) instead of ["key"], because ["key"] can raise KeyError